lib/chutney/TorNet.py

```diff
@@ -763,19 +766,28 @@ def _genRouterKey(self):
                   .format(" ".join(cmdline), stdouterr))
             sys.exit(1)
         self._env['fingerprint'] = fingerprint
-
+       
 
     def _setEd25519Id(self):
-        """Make chutney check for relay microdescriptors before verifying"""
+        """Read the ed25519 identity key for this router, and set up the 'ed25519-id' entry in the Environ"""
         datadir = self._env['dir']
-        key_directory =  os.path.join(datadir, 'keys', "ed25519_master_id_public_key")
-        with open(key_directory ,'rb') as f:
-            f.seek(32)
-            rest_file = f.read()
-            EncodedValue = base64.b64encode(rest_file)
-            ed25519_id = str(EncodedValue)[2:-2]
-            self._env['ed25519-id'] = ed25519_id
-      
+        key_directory = os.path.join(datadir, 'keys', "ed25519_master_id_public_key")
+        if os.path.exists(key_directory):
```

Comment:
I think you're missing a "not" here.

If you look at the continuous integration (CI), you can see the failure here:
https://travis-ci.org/github/torproject/chutney/jobs/664081445#L1568

If you can, please run your code on your local machine, before updating the PR.